ext/include/opentelemetry/ext/zpages/tracez_data.h

```diff
@@ -0,0 +1,137 @@
+#pragma once
+
+#include <iostream>
+#include <list>
+#include <array>
+#include <string>
+
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/nostd/string_view.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+#include "opentelemetry/trace/canonical_code.h"
+#include "opentelemetry/trace/span_id.h"
+#include "opentelemetry/trace/trace_id.h"
+#include "opentelemetry/version.h"
+
+using opentelemetry::sdk::trace::SpanData;
+using opentelemetry::trace::CanonicalCode;
+using opentelemetry::trace::SpanId;
+using opentelemetry::trace::TraceId;
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace ext {
+namespace zpages {
+
+/** Convinient data structure for converting status code enum to string **/
+const std::array<std::string,17> kStatusCodeToString = {
+  "OK",
```

Comment:
nit: const - inited from string literal, it might be nicer to just use **nostd::string_view** here.. But okay.